ios/App/App.xcodeproj/project.pbxproj

```diff
@@ -358,7 +358,7 @@
 				CODE_SIGN_ENTITLEMENTS = App/App.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 1;
+				CURRENT_PROJECT_VERSION = 2;
```

Comment:
I just updated build number for "Capture Exp App" to be able to upload to TestFlight